### DISCUSSION TITLE
Music Playback

### DISCUSSION THREAD

**Discussion by jxmx** [jellyfin-archive/jellyfin-roku-legacy#988] [Q&A]
Hello... 1.6.3 on Roku. Is there a particular set of metadata necessary to allow music to be properly played on the TV? Works on all other clients I have. But on Roku it shows an artist or a playlist, but when I select either the Roku app says there's no songs that match the item I picked.

**Comment by cewert** [comment]
Please make a bug report if you are still experiencing this issue.
